app/controllers/events_controller.rb

```diff
@@ -12,6 +12,9 @@ def index
   # GET /events/1
   def show
     @current_user = current_user
+    @comment = Comment.new
+    @comments = @event.comments.order("created_at DESC")
+    @answer = Answer.new
```

Comment:
Add a guard or fallback error here if any of Comment, Event or Answer are null

Reply:
For security, I already put this line of code on each model comment and answer : "validates :body, presence: true, allow_blank: false" so the comment will not be created if it is empty. 
For the null, an answer depends on a comment and a comment depends on an event, so if the event is null, a comment can't exist, no way to create one. and with the event, in the create method, there is already a condition. 

Translated with www.DeepL.com/Translator (free version)

Reply:
Got it, so seems is ok